async-csharp/RunAsycMethodSynchronously/RunAsycMethodSynchronously/PersonService.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace RunAsyncMethodSynchronously;
+
+public class PersonService
+{
+    public List<Person> GetPeople()
+    {
+        var task = new Task<List<Person>>(() =>
+        {
+            Thread.Sleep(1000);
```

Comment:
For all delays in the Github code, I know you have reduced them to be 1 second but they can be reduced further to 200 (2ms) for instance. 
### **Please, do this just for the Code in Github.  The code in the article should stay with 1 second.**

Reply:
Sorry @aineaTheSecond, One more change and we will be good :)

Reply:
No problem, changes coming up in a minute

Reply:
So the code in the article should be different from what's in the source code on GitHub?

Reply:
Thanks! 
Yes, for this, it's fine. 
We don't want the Nightly build to take a lot of time and I think the readers will get the point.

Reply:
Great!
If there are more changes to be made, just let me know and I'll make the updates.
Thanks!